MapboxAndroidDemo/src/main/AndroidManifest.xml

```diff
@@ -7,7 +7,8 @@
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE"/>
 
     <uses-feature android:name="android.hardware.location.gps"/>
```

Comment:
The chromebook mentioned does have a touchscreen but will probably be missing a GPS, 
could we make that optional as well?